modules/reference/pages/users/user-details.adoc

```diff
@@ -19,4 +64,10 @@ Configure preference settings for users by navigating to the [guimenu]``Preferen
 | CSV Files             | Select whether to use comma or semi-colon delimiters when producing downloadable CSV files. | Comma
 |===
 
-For more information about managing users, see xref:administration:users.adoc[].
+
+
+== Addresses
+
+This tab shows the mailing addresses associated with the user.
```

Comment:
Elsewhere we just say mail address or e-mail address.

Reply:
Oh, is "mailing" an Australianism?